desktop-app/app/components/UserPreferences/index.js

```diff
@@ -106,6 +107,15 @@ export default function UserPreference({
           />
         </div>
       </div>
+      <div className={cx(commonStyles.sidebarContentSectionContainer)}>
```

Comment:
@kirubakarthikeyan 2 small UI things here:

1. Can you add a label to this input field as `Screenshot Location:`.
2. Move this input section a little down, so that there is some space between the checkbox settings and this one. It would be great if you could add a subtle top border to market the separation.

Reply:
I've updated with the neccessary changes, below i've attached the screenshot for the changes i've made.

![screenshot](https://user-images.githubusercontent.com/38885946/87249646-12fab900-c415-11ea-8c21-a787497ffac3.PNG)